src/main/java/com/eprosima/fastcdr/idl/templates/TypesSwigInterface.stg

```diff
@@ -53,6 +53,7 @@ $ctx.directIncludeDependencies : {include | %include "$include$.i"}; separator="
 
 %include <fastcdr/config.h>
 %import(module="fastdds") "fastcdr/xcdr/optional.hpp"
+%import(module="fastdds") "fastcdr/cdr/fixed_size_string.hpp"
```

Comment:
Maybe we could add this line only when there is a string, like it is done [here](https://github.com/eProsima/Fast-DDS-Gen/blob/master/src/main/java/com/eprosima/fastcdr/idl/templates/TypesHeader.stg#L44)

Also do the same with optional like it is done [here](https://github.com/eProsima/Fast-DDS-Gen/blob/master/src/main/java/com/eprosima/fastcdr/idl/templates/TypesHeader.stg#L50)